app/kubernetes-ingress-controller/2.1.x/references/custom-resources.md

```diff
@@ -249,7 +249,7 @@ Please follow the
 [Using the KongIngress resource](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-kongingress-resource)
 guide for details on how to use this resource.
 
-For reference, the following is a complete spec for KongIngress:
+For reference, the following is a complete spec for KongIngress (for properties information see [Upstream][kong-upstream], [Service][kong-service] and [Route][kong-route] entities):
```

Comment:
```suggestion
For reference, the following is a complete spec for KongIngress (for properties information see [Upstream][/gateway/latest/admin-api/#upstream-object], [Service][/gateway/latest/admin-api/#service-object] and [Route][/gateway/latest/admin-api/#route-object] entities):
```

Thank you for this clarification. I've updated the links as we're moving away from link variables. Would you accept this change?

Reply:
Sure I'll!